web_src/js/features/aria.js

```diff
@@ -97,8 +99,9 @@ function attachOneDropdownAria($dropdown) {
     // if the popup is visible and has an active/selected item, use its id as aria-activedescendant
     if (menuVisible) {
       $focusable.attr('aria-activedescendant', $active.attr('id'));
-    } else if (!isComboBox) {
-      // for menu, when the popup is hidden, no need to keep the aria-activedescendant, and clear the active/selected item
+    } else if (!isComboBox && !needKeepAtive) {
+      // for menu in most cases, when the popup is hidden, no need to keep the aria-activedescendant, and clear the active/selected item
+      // if you really need to keep the aria-activedescendant, plz add a className 'keep-active' to the dropdown
```

Comment:
`menu` doesn't keep last selected item, and it would break some screen readers IIRC.

You need to make it a `combobox`.